src/ext/java/org/opentripplanner/ext/emissions/DigitransitEmissionsMode.java

```diff
@@ -0,0 +1,55 @@
+package org.opentripplanner.ext.emissions;
+
+import java.io.Serializable;
+import org.opentripplanner.framework.lang.Sandbox;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+@Sandbox
+public class DigitransitEmissionsMode implements Serializable {
+
+  private String name;
+  private float avg;
+  private int p_avg;
```

Comment:
Underscores(C style) are not allowed in any package, class, field or variable names according to our coding guideline.